experiments.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from points_removal_scripts.open3d_based_script import HPR_open3d
+from points_removal_scripts.mesh_based_script import HPR_mesh_based
+import numpy as np
```

Comment:
Let's sort imports alphabetically and by type.
At first `import ...` and then `from ...`. They must be divided by empty line.
Here it can look like this:
```suggestion
import numpy as np
import sys

from points_removal_scripts.mesh_based_script import HPR_mesh_based
from points_removal_scripts.open3d_based_script import HPR_open3d
```